docs/designs/serving_design.md

```diff
@@ -0,0 +1,101 @@
+# Serving Design
+
+## Background
+
+Model serving is an essential part in an end-to-end machine learning lifecycle. Publishing the trained model as a service in production can make it valuable in the real world. In the literature, we focus on how to save the model trained in ElasticDL for serving.
+
+[SavedModel](https://www.tensorflow.org/guide/saved_model?hl=zh_cn) is the universal serialization format for TensorFlow models. It's language neutral and can be loaded by multiple frameworks (such as TFServing, TFLite, TensorFlow.js and so on). We choose to export the ElasticDL model to the SavedModel format. In this way, we can leverage various mature solutions to serving our model in different scenarios.
```

Comment:
Let's use https://www.tensorflow.org/guide/saved_model for the `SavedModel` link.

Reply:
Good idea, updated